frontend/awx/views/jobs/hooks/useJobHeaderActions.tsx

```diff
@@ -70,7 +72,16 @@ export function useJobHeaderActions(onComplete: (jobs: UnifiedJob[]) => void) {
         ouiaId: 'job-detail-delete-button',
         isDanger: true,
       },
+      {
```

Comment:
Lets put the download button above the delete button and but a separator between the 2 items.

Reply:
done